src/app/modules/item/components/chapter-children/chapter-children.component.html

```diff
@@ -18,8 +18,7 @@
 
     <ng-container *ngIf="itemData && itemData.item.childrenLayout === 'Grid'; else listSection">
       <div
-        class="grid-container"
-        [ngClass]="{ 'full-frame': (fullFrame$ | async)?.active }"
+        class="grid-container full-frame"
```

Comment:
I was done for more flexible setting the size of cards for full and non full mode -> https://github.com/France-ioi/AlgoreaFrontend/commit/534ec5f5b036992f6dc8bda31519a9ac0a38fdb4#diff-15882a5899129eb357146a6893c239a703799d352627ae4a10b54390dcc2e0f8R74

Reply:
Would you have screenshots? When I tested, I didn't see something in particular.

Reply:
As usual

1. Go to this [page](https://dev.algorea.org/branch/bugfix/prevent-scroll-bar-jump-forced-scroll/en/a/899800937596940136;p=;a=0)
2. And I change width of browser
3. Then I see specific settings for size of cards which depends of size of non full frame mode
4. Then I activate full frame mode
5. Then I see another specific settings for size of cards which depends of another size of full frame mode